ingestion/src/metadata/orm_profiler/processor/orm_profiler.py

```diff
@@ -132,6 +141,14 @@ def get_table_profile_sample(self, table: Table) -> Optional[float]:
             if my_record_tests and my_record_tests.profile_sample:
                 return my_record_tests.profile_sample
 
+        if self.workflow_profile_sample:
+            if (
+                table.profileSample is not None
+                and self.workflow_profile_sample != table.profileSample
+            ):
+                return table.profileSample
+            return self.workflow_profile_sample
+
         return table.profileSample or None
```

Comment:
should this last return be just `return None`?

Reply:
It covers the case where you would run your profiler from the CLI without a sample_profile value but assuming it already exists in the table entity.

Reply:
ooh you're right!! thanks for explaining! LGTM